src/App.tsx

```diff
@@ -1,61 +1,164 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
```

Comment:
[CHECKLIST ITEM 5] Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. The `usersFromServer` and `todosFromServer` are imported and used directly in the `App` component, which is good. However, ensure that any child components receive their data through props and do not import data themselves.